pkg/network/ndmz/dualstack.go

```diff
@@ -189,12 +195,24 @@ func (d *DualStack) IP6PublicIface() string {
 // waitIPS waits to receives some IP from DHCP and Router advertisement
 func waitIP4() error {
 	// run DHCP to interface public in ndmz
-	probe := dhcp.NewProbe()
+	probe, err := dhcp.NewBackgroundProbe()
```

Comment:
This is not right. This method should only try to get an IP from DHCP it should not create a service that maintain the DHCP alive. 
Only in the `Create` functions [there](https://github.com/threefoldtech/zos/blob/2f295e39c9b77f57fd6839e68d3c2f1e0f7ad50e/pkg/network/ndmz/dualstack.go#L90) and [there](https://github.com/threefoldtech/zos/blob/2f295e39c9b77f57fd6839e68d3c2f1e0f7ad50e/pkg/network/ndmz/hidden.go#L37)` we need to create the dhcp service only as a last step of the after `waitIP4` and `waitIP6`.